2022q4/riscv.adoc

```diff
@@ -12,22 +12,22 @@ This quarter I resumed work on improvements to FreeBSD's RISC-V architecture sup
 Work was focused primarily on small bug-fixes and improvements, and tooling.
 A handful of known panics and bug reports were fixed and closed.
 
-On the performance tooling side, some issues with the use of DTrace on riscv were found and addressed.
+On the performance tooling side, some issues with the use of DTrace on riscv64 were found and addressed.
```

Comment:
@mhorne: I know nothing about riscv and what has been done with it, could you please confirm if changing riscv with riscv64 is a corret and useful in this case?

Reply:
Confirmed. Either form is acceptable when referencing the FreeBSD architecture support, but riscv64 is slightly more specific so I suggested this one to @grahamperrin.

Reply:
Thanks for your confirmation @mhorne.